Doc/library/pickle.rst

```diff
@@ -716,6 +729,68 @@ share the same dispatch table.  The equivalent code using the
    f = io.BytesIO()
    p = pickle.Pickler(f)
 
+.. _reducer_override:
+
+Subclassing the ``Pickler`` class
+---------------------------------
+
+For most use-cases, it is recommended to simply use the ``dispatch_table`` of a
+``Pickler`` instance to customize its behavior, as explained above.
+
+However, using the ``dispatch_table`` may not be flexible enough. In particular
+we may want to customize the pickling logic based on another criterion than the
+object's type, or we may want to customize the pickling of functions and
+classes.
+
+For those cases, it is possible to subclass from the ``Pickler`` class and
+implement a ``reducer_override`` method. This method can return an arbitrary
```

Comment:
You ca now use:

    :meth:`reducer_override`

instead of:

    ``reducer_override``